packages/nimble-components/src/tooltip/tests/tooltip-matrix.stories.ts

```diff
@@ -110,6 +110,11 @@ export const tooltipLightThemeWhiteBackground: Story = createFixedThemeStory(
     lightThemeWhiteBackground
 );
 
+// Temporarily disabling this test because of flakiness
```

Comment:
Can you update these comments to link to the work item that tracks re-enabling these? #1106

Reply:
Done